### PR TITLE
Fixing metrica epts loader

### PR DESCRIPTION
**Pull request by jcnunezmoreno**
- Frame count name is load from xml file instead of use hardcoded value (#158)
- Including verification before indexing the sensor value to avoid crash  (#157)

### CODE REVIEW

File: kloppy/infra/serializers/tracking/metrica_epts/deserializer.py [hunk 1]
```diff
@@ -54,11 +54,11 @@ def _frame_from_row(
 
                 other_data = {}
                 for sensor in other_sensors:
+                    player_sensor_field_str = f"player_{player.player_id}_{sensor.channels[0].channel_id}"
+                    player_sensor_val = None if player_sensor_field_str not in row else row[player_sensor_field_str]
```
Comment (koenvo) [inline kloppy/infra/serializers/tracking/metrica_epts/deserializer.py:58]: `player_sensor_val = row.get(player_sensor_field_str)`

**Review by koenvo — COMMENTED** [review]
Thanks for this PR!

Could you update a Metrica EPTS test so it will test this behavior? And one minor code improvement comment.

File: kloppy/infra/serializers/tracking/metrica_epts/deserializer.py (hunk 1 above)
Comment (jcnunezmoreno) [reply kloppy/infra/serializers/tracking/metrica_epts/deserializer.py]: Done!

**Comment by koenvo** [comment]
Sorry for the delay. Thanks for the changes! Looking good to me. Can you also run code formatting?

**Comment by jcnunezmoreno** [comment]
> Sorry for the delay. Thanks for the changes! Looking good to me. Can you also run code formatting?

Done! Thanks!!

**Pull request merged**